### ISSUE TITLE
Group label offset too far

### ISSUE THREAD

**Issue by wtfrazr** [Pottus/Texture-Studio#79]
So this was always an issue for me. The purple "Grouped" label is offset by a meter or what and it annoys me cause when I have multiple small objects very close to each other, I cannot determine which object(s) is/are added.

I suggest to add this label to the /edittext3d supported labels and move it to that group as well, maybe on the bottom of the list, but before /note's label.
Please keep the color though, it stands out and I kind of got used to it being purple.

**Comment by Crayder** [comment]
Yeah I tried to make all of the colors stand out compared to the rest. And yeah, good idea.

**Comment by wtfrazr** [comment]
Also the Group label appeared regardless of the tags being toggled or not when you added an object to the group. Can we have that back with the Index label appearing with it as well?

Sometimes it is hard to find out what you actually added with /gsel, cause it is tricky to use but in some cases it is a better choice than /gadd.
